index.html

```diff
@@ -10,14 +10,51 @@
     crossorigin="anonymous"
     href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
     integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==">
+  <link href="https://fonts.googleapis.com/css2?family=Mukta:wght@200;400&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+  <header class="purple-bg">
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h3 class="header-name">Code Journal</h3>
+        </div>
+      </div>
+    </div>
   </header>
   <main>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1 class="new-entry-h">New Entry</h1>
+        </div>
+      </div>
```

Comment:
You should keep this inside the `data-view="entry-form"` element, it should only be visible on that view.